packages/cli/src/cmds/openapi.ts

```diff
@@ -111,6 +145,10 @@ export default {
     args.option('appmap-dir', {
       describe: 'directory to recursively inspect for AppMaps',
     });
+    args.option('max-size', {
+      describe: 'maximum AppMap size that will be processed, in filesystem-reported MB',
```

Comment:
"filesystem-reported" seems like a slightly odd way to describe this: "filesystem-reported" as opposed to what? Maybe this should just be "in MB"?